prime-router/quick-test.sh

```diff
@@ -255,17 +227,53 @@ fi
 # run louisiana
 if [ $RUN_LA -ne 0 ]
 then
-  LA_FILE_SEARCH_STR="/la.*\.csv"
+  LA_FILE_SEARCH_STR="/la.*\.hl7"
   echo Generate synthetic LA data, HL7!
-  text=$(./prime data --input-fake 50 --input-schema covid-19 --output-dir $outputdir --target-states LA --output-format HL7_BATCH)
-  parse_prime_output_for_filename "$text" "/covid"
+  text=$(./prime data --input-fake 50 --input-schema la/la-covid-19 --output-dir $outputdir --target-states LA --output-format HL7_BATCH)
+  parse_prime_output_for_filename "$text" "$LA_FILE_SEARCH_STR"
+fi
 
-  echo Now send that fake LA data through the router
-  # TODO: can we import HL7?
-  # TODO: how do we compare HL7?
 
-  echo Now those _those_ LA results back in to their own schema and export again!
-  # TODO: once we've imported HL7 we can finish this step
+# run florida
+if [ $RUN_FL -ne 0 ]
+then
+  FL_FILE_SEARCH_STR="/fl.*\.hl7"
+  # FLORIDA, MAN
+  echo Generate fake FL data
+  text=$(./prime data --input-fake 50 --input-schema fl/fl-covid-19 --output-dir $outputdir --target-states FL --target-counties Broward --output-format HL7_BATCH)
+  parse_prime_output_for_filename "$text" $FL_FILE_SEARCH_STR
+fi
+
+# run north dakota
+if [ $RUN_ND -ne 0 ]
+then
+  echo Generate fake ND data, HL7!
+  text=$(./prime data --input-fake 50 --input-schema nd/nd-covid-19 --output-dir $outputdir --target-states ND --target-counties Richland --output-format HL7_BATCH)
+  parse_prime_output_for_filename "$text" "/nd.*\.hl7"
+fi
+
+# run tx
+if [ $RUN_TX -ne 0 ]
+then
+  echo Generate fake TX data, HL7!
+  text=$(./prime data --input-fake 50 --input-schema tx/tx-covid-19 --output-dir $outputdir --target-states TX --target-counties Knox --output-format HL7_BATCH)
+  parse_prime_output_for_filename "$text" "/tx.*\.hl7"
+fi
+
+# run vt
+if [ $RUN_VT -ne 0 ]
+then
+  echo Generate fake VT data, HL7!
+  text=$(./prime data --input-fake 50 --input-schema vt/vt-covid-19 --output-dir $outputdir --target-states VT --target-counties Essex --output-format HL7_BATCH)
+  parse_prime_output_for_filename "$text" "/vt.*\.hl7"
+fi
+
+# run nm
+if [ $RUN_NM -ne 0 ]
+then
+  echo Generate fake NM data, HL7!
+  text=$(./prime data --input-fake 50 --input-schema nm/nm-covid-19 --output-dir $outputdir --target-states NM --target-counties Hidalgo --output-format HL7_BATCH)
+  parse_prime_output_for_filename "$text" "/nm.*\.hl7"
 fi
```

Comment:
I reorganized the calls as well, and updated to use `target-states` and `target-counties`, as well as use the state-specific schemas now.

Reply:
Once you've got hl7 ingest working via the cli, then you can do the 'eat our own dog-schemas' test where you read it right back in again, that has been great at finding (creating?) obscure bugs.